web/src/compose_state.js

```diff
@@ -88,20 +86,7 @@ export function stream_name() {
     return "";
 }
 
-export function set_stream_name(stream_name) {
-    if (!stream_name) {
-        compose_recipient.set_selected_recipient_id("");
-        return;
-    }
-
-    // If we fail to select a stream that the caller expects
-    // us to do successfully, we should throw an error.
-    const stream_id = stream_data.get_stream_id(stream_name);
-    if (stream_id === undefined) {
-        blueslip.error("Unable to select stream: " + stream_name);
-        compose_recipient.set_selected_recipient_id("");
-        return;
-    }
+export function set_stream_id(stream_id) {
```

Comment:
This is just a wrapper around a single function call, but I figured it would be nice to keep a function at the `compose_state` level.